src/Management/src/Endpoint/Actuators/HttpExchanges/ConfigureHttpExchangesEndpointOptions.cs

```diff
@@ -81,28 +82,26 @@ public override void Configure(HttpExchangesEndpointOptions options)
         // It's not possible to distinguish between null and an empty list in configuration.
         // See https://github.com/dotnet/extensions/issues/1341.
         // As a workaround, we interpret a single empty string element to clear the defaults.
-        if (options.RequestHeaders.Count == 0)
+        if (options.RequestHeaders.Count == 0 || !string.IsNullOrEmpty(options.RequestHeaders[0]))
```

Comment:
We discussed this and decided that anything in configuration should be additive to the default whitelist. Note this makes it _impossible_ to exclude entries from the default whitelist _from configuration_. Although unlikely to ever be needed, it can be accomplished by registering a (post)configurer.